main.py

```diff
@@ -0,0 +1,8 @@
+import numpy as np
+from patblib import Path
+
+
+def z(x, y):
```

Comment:
What does z mean? What are x and y? Consider adding some documentation here (maybe type hinting too) so that someone who is unfamiliar with this codebase can understand what you're doing.

Reply:
I added documentation here 45cd48dd64bae55521182d52a6cf225c0d39076b